app/javascript/utils/react-bootloader.jsx

```diff
@@ -140,6 +132,16 @@ export function renderComponents(parentElement, mappings) {
   }
 }
 
+function renderTooltips(parentElement, mappings) {
+  if (!parentElement) {
+    parentElement = document.body
+  }
+
+  parentElement
+    .querySelectorAll('[data-tooltip-type][data-endpoint]')
+    .forEach((elem) => renderTooltip(mappings, elem))
+}
+
```

Comment:
I just flipped the order of this function with the `renderTooltip` one. You can ignore this and the related change below.